compiler/tests/testSuite.bal

```diff
@@ -57,6 +76,26 @@ function testCompileEU(string path, string kind) returns file:Error|io:Error? {
     }
 }
 
+function importFilename(string path, string errFilename) returns string? {
+    int? index = path.lastIndexOf(".bal");
+    if index is int {
+        string subModPath = path.substring(0, index) + ".modules";
+        if checkpanic file:test(subModPath, file:EXISTS) && checkpanic file:test(subModPath, file:IS_DIR) {
+            foreach var subMod in checkpanic file:readDir(subModPath) {
+                if !checkpanic file:test(subMod.absPath, file:IS_DIR) {
+                    continue;
+                }
+                foreach var test in checkpanic file:readDir(subMod.absPath) {
+                    if  checkpanic file:basename(test.absPath) == errFilename {
+                        return test.absPath;
+                    }
+                }
+            }
+        }
+    }
+    return ();
```

Comment:
return nil is implicit, no need to type it.

Reply:
fixed